roles/partition-disk/tasks/extend_root_partition.yaml

```diff
@@ -0,0 +1,16 @@
+  - name: "Extend root partition LV to all available VG space"
+    when: item.key in "lv_root"
+    lvol:
+      vg: "{{ item.value.vg }}"
+      lv: "{{ item.key }}"
+      size: +100%FREE
+    with_dict: "{{ ansible_lvm.lvs }}"
+
+  - name: "Resize the root filesystem"
+    when: item.key in "lv_root"
+    filesystem:
+      dev: "/dev/{{ item.value.vg }}/{{ item.key }}"
+      fstype: xfs
+      resizefs: yes
+    with_dict: "{{ ansible_lvm.lvs }}"
+...
```

Comment:
What are the three dots here?

Reply:
I use that for YAML files to mark end of the file, as I had read about it here, https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics